docs/03_server/05_access-control/03_password_authentication.md

```diff
@@ -427,20 +488,30 @@ These services should be contacted on the hosts in the order they are defined in
     DETAILS.SERVICE[1].HOST[1].NAME = genesisserv2
     DETAILS.SERVICE[1].HOST[1].PORT = 9002
 
+### Change password
+A password can expire in three different scenarios: time based, user based or admin based. The end result is always the same, the user will need to change the password on the next login, as their previous one has now expired.
+
+The password expiry mechanism can be triggered by sending a message like this:
+
+    MESSAGE_TYPE = EVENT_EXPIRE_USER_PASSWORD
+    DETAILS.USER_NAME = JohnWolf
+
+It is common for administrators to help users recover their account credentials by expiring the current User record `STATUS` with a new one-time password, and this action will force users to change it on their first login. In this case the message can optionally receive a one-time password like shown below:
+
+    MESSAGE_TYPE = EVENT_EXPIRE_USER_PASSWORD
+    DETAILS.USER_NAME = JohnWolf
+    DETAILS.PASSWORD = ******
 
 ### Rights polling
-The GUI can receive rights from a process called `AUTH_DATASERVER`. The view `USER_RIGHTS` displays all users and codes. A logged-in user should automatically set the Filter expression to be `USER_NAME`=='xxx' to receive push updates to user privileges.
+The GUI can receive rights from a process called `GENESIS_AUTH_DATASERVER`. The view `ALL_USER_RIGHTS` displays all users and codes. A logged-in user should automatically set the Filter expression to be `USER_NAME`=='xxx' to receive push updates to user privileges.
 
 ## Entity management
 In the Genesis low-code platform, there are profiles, users and rights. A profile is a group of users, which can be permissioned. For example, you could have a SALES_TRADER group in which all users must have the same permissions. In all cases where you specify either a right for a user/profile, or a user in a profile, the event represents what you want the entity to look like; i.e. if you amend a profile and don't supply a user that previously was part of that profile, then that user will be removed from that profile on the server.
 
 Note the following:
 
-* 2-phase validation is not currently supported
-* metadata is not supported on the following transactions.
-  User/profile `STATUS` field can be set to `ENABLED`, `DISABLED`, `PASSWORD_EXPIRED` and `PASSWORD_RESET`.
-  `PASSWORD_EXPIRED` should prompt the user to enter a new password.
-  `PASSWORD_RESET` should do the same but the server expects a blank "current password" field.
+* User `STATUS` field can be set to `ENABLED`, `DISABLED` or `PASSWORD_EXPIRED`. A User set up with `PASSWORD_EXPIRED` should prompt the user to enter a new password on next login.
```

Comment:
The information provided in the old docs was correct for many versions ago, but it is incorrect as of now. We do support 2-phase validation and we also provide metadata now.